src/react/components/_dashboard/NotificationsCard/MessagesBySource.js

```diff
@@ -1,38 +1,78 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 
 import SourcedMessages from './SourcedMessages';
 
+const sum = (acc, current) => {
+  return acc + current;
+};
+
 const MessagesBySource = ({
   groupedNotifications,
   selectedSource,
   expandedItem,
   setExpandedItem,
   hasFocus,
-}) => (
-  <>
-    {groupedNotifications.map(dateGroup =>
-      dateGroup.messagesBySource.map((source, index) => {
-        if (selectedSource === '' || source.sourceName === selectedSource) {
-          return (
-            <SourcedMessages
-              key={index}
-              date={dateGroup.date}
-              source={source.sourceName}
-              type={source.type}
-              messages={source.messages}
-              expandedItem={expandedItem}
-              setExpandedItem={setExpandedItem}
-              hasFocus={hasFocus}
-            />
-          );
-        } else {
-          return null;
-        }
-      })
-    )}
-  </>
-);
+}) => {
```

Comment:
@developish - What is this syntax change about? What's the difference?

Reply:
When it's just the parenthesis it's returns directly (AKA an implicit return). I had to do some additional logic for the "Show More" stuff, so I had to open a block and do an explicit return of the component